.env

```diff
@@ -2,7 +2,7 @@ REACT_APP_BUILD_DATETIME=0
 REACT_APP_BUILD_SHA=sha0000000000000000000000000000000000000
 REACT_APP_BUILD_VERSION=
 # REACT_APP_API=http://localhost:8008/v1/
-REACT_APP_API=https://nexus.stg.oasis.io/v1/
-REACT_APP_TESTNET_API=https://testnet.nexus.stg.oasis.io/v1/
-# REACT_APP_API=https://nexus.oasis.io/v1/
-# REACT_APP_TESTNET_API=https://testnet.nexus.oasis.io/v1/
+# REACT_APP_API=https://nexus.stg.oasis.io/v1/
```

Comment:
revert ?

Reply:
It's in a separate commit, which I intend to drop before merging.

Reply:
ahh Ok